app/Jobs/ApplicationDeploymentJob.php

```
@@ -715,9 +737,16 @@ private function deploy_docker_compose_buildpack()
         $server_workdir = $this->application->workdir();
         if ($this->application->settings->is_raw_compose_deployment_enabled) {
             if ($this->docker_compose_custom_start_command) {
+                // Auto-inject -f (compose file) and --env-file flags using helper function
+                $start_command = injectDockerComposeFlags(
```

PHPMD (2.15.0)

The variable $start_command is not named in camelCase. (undefined)

(CamelCaseVariableName)